pkg/jobs/hive/hive.go

```diff
@@ -335,11 +336,32 @@ func UpgradeCluster(client clientv1.Client, clusterName string, curator *cluster
 		return getErr
 	}
 
-	if cvAvailableUpdates, ok := clusterVersion["status"].(map[string]interface{})["availableUpdates"].([]interface{}); ok {
-		for _, version := range cvAvailableUpdates {
-			if version.(map[string]interface{})["version"] == desiredUpdate {
-				clusterVersion["spec"].(map[string]interface{})["desiredUpdate"] = version
-				break
+	curatorAnnotations := curator.GetAnnotations()
+
+	if curatorAnnotations != nil && curatorAnnotations[ForceUpgradeAnnotation] == "true" {
+		// Usually when the desired version is in availableUpates we use the image hash from there
+		// but for non-recommended images we don't have the hash so we have to use the image with
+		// a version tag instead
+		if clusterVersion["spec"].(map[string]interface{})["desiredUpdate"] != nil {
+			clusterVersion["spec"].(map[string]interface{})["desiredUpdate"].(map[string]interface{})["version"] = desiredUpdate
+			clusterVersion["spec"].(map[string]interface{})["desiredUpdate"].(map[string]interface{})["force"] = true
+			clusterVersion["spec"].(map[string]interface{})["desiredUpdate"].(map[string]interface{})["image"] =
+				"quay.io/openshift-release-dev/ocp-release:" + desiredUpdate + "-multi"
+		} else {
+			// For when desiredUpdate does not exist
+			clusterVersion["spec"].(map[string]interface{})["desiredUpdate"] = map[string]interface{}{
```

Comment:
```
clusterVersion["spec"].(map[string]interface{})["desiredUpdate"]
```
Minior nit. Up to you. Let's put this in a variable so it's easier to read since it's been used so many times.

Reply:
Sounds good! Let me fix this.